tools/runners/run-test-suite-test262.sh

```diff
@@ -52,23 +52,30 @@ fi
 rm -rf "${PATH_TO_TEST262}/test/suite/bestPractice"
 rm -rf "${PATH_TO_TEST262}/test/suite/intl402"
 
-# TODO: Enable these tests after daylight saving calculation is fixed.
-rm -f "${PATH_TO_TEST262}/test/suite/ch15/15.9/15.9.3/S15.9.3.1_A5_T1.js"
-rm -f "${PATH_TO_TEST262}/test/suite/ch15/15.9/15.9.3/S15.9.3.1_A5_T2.js"
-rm -f "${PATH_TO_TEST262}/test/suite/ch15/15.9/15.9.3/S15.9.3.1_A5_T3.js"
-rm -f "${PATH_TO_TEST262}/test/suite/ch15/15.9/15.9.3/S15.9.3.1_A5_T4.js"
-rm -f "${PATH_TO_TEST262}/test/suite/ch15/15.9/15.9.3/S15.9.3.1_A5_T5.js"
-rm -f "${PATH_TO_TEST262}/test/suite/ch15/15.9/15.9.3/S15.9.3.1_A5_T6.js"
+echo "Starting test262 testing for ${ENGINE}. Running test262 may take several minutes."
 
-echo "Starting test262 testing for ${ENGINE}. Running test262 may take a several minutes."
+function progress_monitor() {
+  NUM_LINES_EXPECTED=11558
```

Comment:
This is too hard-wired. How did you get this number? Why can't it be computed here by this script? (BTW, summary prints me 11552 executed tests.)

Reply:
This is number of lines, not number of tests, so it is slightly more than the number of tests. Previously it just ran for several minutes without giving any feedback, slightly wrong feedback (like if the test suite changes) seems a lot better than no feedback to me. I think the right place to put this is probably down a level in `test262.py`, but that is a considerably more involved change that I'm not about to attempt.

I suppose you could always just print number of tests executed, and not the percentage... at least then you can see that it's progressing. It still seems worse than giving the percentage, even if it's slightly off, but maybe it's an approach that will age better.